roles/openshift_docker_facts/tasks/main.yml

```diff
@@ -65,6 +65,6 @@
 
 - name: Set docker version to be installed
   set_fact:
-    docker_version: "{{ '1.9.1' }}"
+    docker_version: "{{ '1.10.3' }}"
```

Comment:
True, extending that logic into ansible is something we should try to avoid. Which would mean having to deal with some yum shell shenanigans or such...

The complication here seems to be around containerized deployments, though...  Maybe we need to segregate these types of checks there and let the RPM dependencies manage it for us in the case of non-containerized installations?